src/test/java/org/broadinstitute/hellbender/tools/walkers/annotator/VariantAnnotatorIntegrationTest.java

```diff
@@ -1,342 +1,369 @@
+            Assert.assertEquals(keyForVariant(mutect2VC), keyForVariant(reannotatedVC));
+
+            // check INFO field annotations
+            representativeInfoBooleanAnnotations.stream()
+                    .flatMap(a -> a.getKeyNames().stream())
+                    .forEach(key -> Assert.assertEquals(mutect2VC.hasAttribute(key), reannotatedVC.hasAttribute(key)));
+
+            representativeInfoIntegerAnnotations.stream()
+                    .flatMap(a -> a.getKeyNames().stream())
+                    .forEach(key -> {
+                                final double[] mutect2Values = Doubles.toArray(mutect2VC.getAttributeAsDoubleList(key, 1.0));
+                                final double[] reannotatedValues = Doubles.toArray(reannotatedVC.getAttributeAsDoubleList(key, 1.0));
+                        (approximatelyEqual(mutect2Values, reannotatedValues) ? matchingInfoFieldValues : nonMatchingInfoFieldValues).increment();
+                            });
+
+            representativeInfoStringAnnotations.stream()
+                    .flatMap(a -> a.getKeyNames().stream())
+                    .forEach(key -> Assert.assertEquals(mutect2VC.getAttribute(key), reannotatedVC.getAttribute(key)));
+
+            // check FORMAT annotations
+            final Genotype mutect2Genotype = mutect2VC.getGenotype(0);
+            final Genotype reannotatedGenotype = reannotatedVC.getGenotype(0);
+
+            representativeGenotypeAnnotations.stream()
+                    .flatMap(a -> a.getKeyNames().stream())
+                    .forEach(key -> {
+                        final double[] mutect2Values = GATKProtectedVariantContextUtils.getAttributeAsDoubleArray(mutect2Genotype, key, () -> null, 1.0);
+                        final double[] reannotatedValues = GATKProtectedVariantContextUtils.getAttributeAsDoubleArray(reannotatedGenotype, key, () -> null, 1.0);
+                        (approximatelyEqual(mutect2Values, reannotatedValues) ? matchingFormatFieldValues : nonMatchingFormatFieldValues).increment();
+                    });
         }
-        args.addOutput(output).addArgument("V", input.getAbsolutePath());
-
 
-        // Handling a difference in syntax between GATK3 and GATK4 wrt. annotation groups
-        additionalArguments = additionalArguments.stream().map(a -> a.contains("Standard") ? a + "Annotation" : a).collect(Collectors.toList());
-        additionalArguments.forEach(args::add);
-
-        Utils.resetRandomGenerator();
-        runCommandLine(args);
-
-        assertHeadersMatch(output, expected, HEADER_LINES_TO_IGNORE);
-        final List<VariantContext> expectedVC = getVariantContexts(expected);
-        final List<VariantContext> actualVC = getVariantContexts(output);
-        assertForEachElementInLists(actualVC, expectedVC, assertion);
+        Assert.assertTrue(matchingInfoFieldValues.intValue() > 40 * nonMatchingInfoFieldValues.intValue());
```

Comment:
What are these seemingly arbitrary constants?

Reply:
Added a comment.  They're the baseline of current accuracy in order to flag future regressions.